api_docs/writing-bots.md

```diff
@@ -17,23 +17,35 @@ On this page you'll find:
 
 ## Installing a development version of the Zulip bots package
 
-1. `git clone https://github.com/zulip/python-zulip-api.git` - clone the [python-zulip-api](
-  https://github.com/zulip/python-zulip-api) repository.
+{start_tabs}
 
-2. `cd python-zulip-api` - navigate into your cloned repository.
+1. Clone the [python-zulip-api](https://github.com/zulip/python-zulip-api)
+   repository:
 
-3. `python3 ./tools/provision` - install all requirements in a Python virtualenv.
+        git clone https://github.com/zulip/python-zulip-api.git
```

Comment:
In general we have a practice of using fenced code blocks in the API documentation; they're a lot easier to reason about their indentation levels in relation to bulleted lists.